src/pages/verify/e-ids/danish-mitid-erhverv.mdx

```diff
@@ -55,6 +55,14 @@ You can determine which kind of option the user selected by inspecting the claim
 - Company signatory logins will have a `companySignatory` claim with value `true`.
 - Private logins will have neither of the above.
 
+### Pefill VAT ID (CVR number)
+
+If you already have an expectation of what company the user will login on behalf of, you can prefill the VAT ID (CVR number)
+to limit their selection choice (if multiple roles in the same company) or skip the company selector all together.
+
+- When activating MitID Erhverv via acr_values: `urn:grn:authn:dk:mitid:business`: `login_hint=vatid:DK<CVR>`
+- When activating MitID Erhverv via login_hint: `login_hint=business vatid:DK<CVR>`
+
```

Comment:
We have a dedicated prefill-section already (https://docs.criipto.com/verify/guides/prefilled-fields/).

Maybe move the contents of this PR to there, and then link to it from here ?

Reply:
Considered it - but this seems very specific to MitID Ervherv, as a reader I would expect to be able to find it here.